abs.yml

```diff
@@ -0,0 +1 @@
+aggregate_branch: 3.12
```

Comment:
@jezdez Could you rename `abs.yml` to `abs.yaml`?

Reply:
done

Reply:
@cbouss You hit me)